### PR TITLE
Valid states for `runtimeStatus`

### LINKED ISSUE #3904: Valid workflow states needs update
**Describe the issue**
The workflow API docs have outdated/incomplete information about valid states of a workflow.

**URL of the docs**
[https://docs.dapr.io/reference/api/workflow_api/#response-content-6](https://docs.dapr.io/reference/api/workflow_api/#response-content-6)

**Expected content**
Valid values for `runtimeStatus` are:
 - `"RUNNING"`
 - `"COMPLETED"`
 - `"CONTINUED_AS_NEW"`
 - `"FAILED"`
 - `"CANCELED"`
 - `"TERMINATED"`
 - `"PENDING"`
 - `"SUSPENDED"`

**Screenshots**
Screenshot 2023-12-14 at 12 01 02

**Additional context**

### PR DESCRIPTION
**Pull request by hhunter-ms**
## Description

Added the valid states for workflow `runtimeStatus`

## Issue reference

PR will close: #3904

### CODE REVIEW

File: daprdocs/content/en/reference/api/workflow_api.md [hunk 1]
```diff
@@ -235,7 +235,7 @@ The API call will provide a JSON response similar to this:
 
 Parameter | Description
 --------- | -----------
-`runtimeStatus` | The status of the workflow instance. Values include: `RUNNING`, `TERMINATED`, `PAUSED`  
+`runtimeStatus` | The status of the workflow instance. Values include: `"RUNNING"`, `"COMPLETED"`, `"CONTINUED_AS_NEW"`, `"FAILED"`, `"CANCELED"`, `"TERMINATED"`, `"PENDING"`, `"SUSPENDED"`
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/workflow_api.md:238]: I presume this is "CANCELLED" and not "CANCELED"?

Comment (hhunter-ms) [reply daprdocs/content/en/reference/api/workflow_api.md:238]: TIL from Grammarly:

> Canceled or cancelled is the past tense of the verb to cancel. Both spellings are correct; Americans favor canceled (one l), while cancelled (two l's) is preferred in British English and other dialects. Canceling/cancelling and canceler/canceller follow the same pattern.

So either way works for me haha

Comment (ItalyPaleAle) [reply daprdocs/content/en/reference/api/workflow_api.md:238]: We use "CANCELED" in the code: https://github.com/microsoft/durabletask-go/blob/119b361079c45e368f83b223888d56a436ac59b9/internal/protos/orchestrator_service.pb.go#L59

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**